keystore/keystore.go

```diff
@@ -189,3 +189,13 @@ type KeyStore interface {
 	GetAuthKey(remove bool) ([]byte, error)
 	Reset()
 }
+
+// MultiKeyStore is a KeyStore with additional interface to retrieve historical versions of the key.
+// It is used to decrypt AcraStructs with possibly rotated keys.
+type MultiKeyStore interface {
+	KeyStore
+
+	GetZonePrivateKeys(id []byte) ([]*keys.PrivateKey, error)
+
+	GetServerDecryptionPrivateKeys(id []byte) ([]*keys.PrivateKey, error)
+}
```

Comment:
**keystore-v2** branch has refactored key access interfaces a bit and that makes this interface unnecessary. These methods can be simply moved into `PrivateKeyStore` interface.

I think it's okay to merge it into master like this to avoid unnecessary changes. I'll sync and clean this up in v2, along with other updates related to multikeys.

Reply:
Okay, left as is